### ISSUE TITLE
Big integer field schema build issue

### ISSUE THREAD

**Issue by pepijn-vanvlaanderen** [relate-app/gatsby-source-strapi#6]
When having a field with type big integer, the schema building breaks:

```
Missing onError handler for invocation 'building-schema', error was 'Error: UnionError[StrapiPageSectionsDynamicZone]: Type with name "Long" does not exists'. Stacktrace was
'Error: UnionError[StrapiPageSectionsDynamicZone]: Type with name "Long" does not exists
```

**Comment by timbrandin** [comment]
Ah, we need to transform this type to another type that Gatsby supports, similar to how we transform the Date type.

**Comment by timbrandin** [comment]
According to [this thread](https://stackoverflow.com/questions/62893664/how-can-i-handle-long-int-with-graphql) we should be able to use Float, will set that for now. Please add a PR if you find another type that is more suitable.

**Comment by timbrandin** [comment]
Released in `4.3.3`
